src/main/java/nearsoft/academy/bigdata/recommendation/MovieRecommender.java

```diff
@@ -30,57 +30,63 @@
     private int reviews = 0;
```

Comment:
I know you're new on Java, so here's a tip: In classes, there's a default value when a class object is created, for `int` types, default value is `0`. I'd like that you investigate all other default values (for primitives and object references)